lib/google_calendar.rb

```diff
@@ -33,7 +33,7 @@ def self.open(*args, &block)
     instance = new(*args)
     block.call(instance)
   ensure
-    instance.cleanup!
+    instance.cleanup! if instance
```

Comment:
You could also say `instance&.cleanup!`.

Reply:
Good point, for some reason I always think the `&.` operator is a Ruby 2.5 feature.